content/en/blog/recap-2022/index.md

```diff
@@ -0,0 +1,137 @@
+
+- Whether an external entity, or an insider, cybercriminals are most commonly abusing services such as Kubernetes and GitHub actions by infecting hosts and containers with cryptojackers and using the business resources to mine cryptocurrency on the attacker's behalf. [Learn how Falco detects cryptominers activity in your cluster](https://falco.org/blog/falco-detect-cryptomining/). 
+
+- The new BPF probe landed among us 👽 this fall with new shiny features. The BPF world grows continuously and every new kernel release introduces some unbelievable novelties! To take advantage of these the community created a completely new architecture, new BPF programs and maps. The main goal was to improve performance, maintainability, and user experience, shipping a unique, powerful, self-contained Falco executable. Discover the modern world of EPF with Falco, read the [full blog here](https://falco.org/blog/falco-modern-bpf/). 
+
+- At re:Invent 2022, it was a pleasure to announce that Falcosidekick would be available with preview integration for Amazon Security Lake, a new service that optimizes and centralizes security data from cloud, on-premises, and custom sources into a purpose-built data lake. Falcosidekick forwards Falco events into other services: the new integration exports security events using the Open Cybersecurity Schema Framework (OCSF) format, an open industry standard, and sends them directly to Amazon Security Lake. This makes it easier to normalize and combine Falco events with other security data sources. You can check out the integration in the next version of Falcosidekick, 2.27.0. Read the full announcement, [Support for Amazon Security Lake](https://falco.org/blog/falco-on-aws/).
+
+- As the home of such important assets, GitHub repositories should be at the top of your list of security priorities. However, many people fail to put in place even basic measures to protect source code repositories. [Read abou](https://falco.org/blog/falco-plugin-github/)t three risks to GitHub repos and how you can reliably detect them, as they happen, using Falco open source security. 
+
+- Before 0.32.1, Falco could not work with gVisor monitored sandboxes because it was not possible to install a kernel module or eBPF probe in such an environment. But wouldn't it be great to leverage the stream of system call information that gVisor collects through its powerful monitoring system directly in Falco? This is exactly what became possible with gVisor release 20220704.0 and Falco 0.32.1. [Learn how to integrate gVisor and Falco on Docker](https://falco.org/blog/intro-gvisor-falco/). 
+
+- With Falco 0.32.1, you can now run Falco on Apple ARM M1 CPUs. It requires a Linux virtual machine (VM) since Falco doesn't run on OSX, but it is pretty straightforward. Here are the step by step [instructions.](https://falco.org/blog/falco-apple-silicon/) 
```

Comment:
I think the big news is support for ARM in general, I'd eventually mention the Apple Silicon compatibility as a +1 example.